Scripts/script-RunDockerCommand.yml

```diff
@@ -0,0 +1,55 @@
+commonfields:
+  id: 9ffdaa61-efdd-47e6-8a88-ea86c949274f
+  version: 2
+name: RunDockerCommand
+script: |-
+  # The command demisto.command() holds the command sent from the user.
+  import subprocess
+  ret = []
+  sysargs = None;
+
+  cmdarg = demisto.args()['cmd']
+  if demisto.args().has_key('sysargs'):
+      sysargs = demisto.args()['sysargs']
+
+  if sysargs == None:
+      output = None
+      try:
+          output = subprocess.check_output(cmdarg, shell=True)
+      except subprocess.CalledProcessError as e:
+          output = e.output
+      ret.append(output)
+  else:
+      ret.append(subprocess.check_output([cmdarg, sysargs]))
+
+
+  ec = {'Command': cmdarg, 'Results': ret[0]}
+  demisto.results({'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': ret, 'HumanReadable': ret[0], 'EntryContext': {'CommandResults': ec}})
```

Comment:
prettify the json 
`{'ContentsFormat': formats['json'],'Type': entryTypes['note'],'Contents': hosts_json,
      'HumanReadable':res,'EntryContext': ec}` => 
```
{
    'Type': entryTypes['note'],
    'ContentsFormat': formats['json'],
    'Contents': hosts_json,
    'HumanReadable':res,
    'ReadableContentsFormat': formats['markdown'],
    'EntryContext': ec
}

```

Reply:
done